components/site/src/link_checking.rs

```diff
@@ -152,7 +157,12 @@ pub fn check_external_links(site: &Site) -> Vec<String> {
     // the ones that have invalid URLs
     for (file_path, links) in external_links {
         for link in links {
-            if should_skip_by_prefix(link, &site.config.link_checker.skip_prefixes) {
+            if should_skip_by_prefix(link, &site.config.link_checker.skip_prefixes)
+                || should_skip_by_file(
+                    file_path,
+                    site.config.link_checker.ignored_files_globset.as_ref().unwrap(),
```

Comment:
can you unwrap before the loop so it's easier to read?

Reply:
done